[BZ2088757] Add note for ODF (#46564)
openshift/openshift-docs: modules/persistent-storage-csi-snapshots-create.adoc

----
+
<1> The name of the CSI driver that is used to create snapshots of this `VolumeSnapshotClass` object. The name must be the same as the `Provisioner` field of the storage class that is responsible for the PVC that is being snapshotted.
+
[NOTE]
====

> Depending on the driver you have used to configure persistent storage, additional parameters might be required or you can use an existing `VolumeSnapshotClass`.

Reviewer: @SNiemann15 Added a suggestion to simplify grammar. 

```suggestion
Depending on the driver that you used to configure persistent storage, additional parameters might be required. You can also use an existing `VolumeSnapshotClass` object.
```

Can merge if "`VolumeSnapshotClass`" -> "`VolumeSnapShotClass` object", as that's in our in-house guidelines. 

Non-blocking questions that I had as I skimmed this change:

* Which drivers cause this behavior?
* Which parameters? Driver-specific ones?
* Is this saying that you can specify the object name in lieu of a driver name? If so, why not update line 41 to say this?
